modules/openapi-generator/src/main/resources/kotlin-server/libraries/ktor/Paths.kt.mustache

```diff
@@ -18,7 +18,7 @@ object Paths {
      {{#allParams}}* @param {{paramName}} {{description}} {{^required}}(optional{{#defaultValue}}, default to {{{.}}}{{/defaultValue}}){{/required}}
      {{/allParams}}*/
     @KtorExperimentalLocationsAPI
-    @Location("{{path}}") class {{operationId}}({{#allParams}}val {{paramName}}: {{{dataType}}}{{#hasMore}}, {{/hasMore}}{{/allParams}})
+    @Location("{{path}}") class {{operationId}}({{#allParams}}val {{paramName}}: {{{dataType}}}{{^required}}? = null{{/required}}{{#hasMore}}, {{/hasMore}}{{/allParams}})
```

Comment:
I think required would have to append `?` if it's nullable, but this would change assignment in the method body not just in the parameter list. Also, rather than assigning `null` I think we'd want to honor defaults. For example, `i: Int? = 1` would be a valid argument.

Reply:
I think this is a bit dangerous, because is a valid value that can be used without the user noticing.